hw3/tests/test_hw3_task01.py

```diff
@@ -0,0 +1,45 @@
+from hw3.hw3_task01 import TestPlug, cache_factory
+
+
+@cache_factory(times=3)
+def f(x):
+    return x * 1.1
+
+
+@cache_factory(times=2)
+def txt(text: str):
+    return text * 2
```

Comment:
Move declaration into the test.

Reply:
I'm not sure what do you mean by this...

Reply:
```python
def test_...
  @cache_factory(times=2)
  def txt(text: str):
      return text * 2

```